cardano-sl.cabal

```diff
@@ -115,8 +113,26 @@ library
                         Pos.Util.UserSecret
 
                         -- Wallets
+                        Pos.Wallet
                         Pos.Wallet.KeyStorage
+                        Pos.Wallet.SscType
+                        Pos.Wallet.WalletMode
+                        Pos.Wallet.Web.ClientTypes
+                        Pos.Wallet.Web.Error
+                        Pos.Wallet.Web.Error.Types
+                        Pos.Wallet.Web.Error.Util
                         Pos.Wallet.Web.Secret
+                        Pos.Wallet.Web.State
+                        Pos.Wallet.Web.State.Acidic
+                        Pos.Wallet.Web.State.State
+                        Pos.Wallet.Web.State.Storage
+                        Pos.Wallet.Web.Util
```

Comment:
@gromakovsky `Pos.Client.Txp.Balances` depends on `Pos.Wallet.Web.State` (see https://github.com/input-output-hk/cardano-sl/blob/5b7acc1d68ad72b5f31acb6e5709eb4c070af20e/src/Pos/Client/Txp/Balances.hs#L27-L28). And with this all others of `Pos.Wallet.Web.State.*` are imported by `Pos.Wallet.Web.State`